share/doc/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ if(SPHINX_FOUND)
     )
     list(APPEND SPHINX_DEPS ${VOTCA_SPHINX_DIR}/${_FILE})
   endforeach()
+  file(COPY _static DESTINATION ${VOTCA_SPHINX_DIR})
```

Comment:
Usually I would create a `add_custom_command(OUTPUT ...`, but I assume we basically need edit the files in `static/`, right?

Reply:
I think that a `custom_command` is then a better option because sphinx does not touch anything inside `_static`, it just reads it to generate the HTML.  we are just copying it to a location where the `conf.py` can find it, and a `_static` file is the convention.

Reply:
So I wasn't clear, I think `COPY` is ok here! `add_custom_command()` is better for the files that gets edited a lot, so that `make` knows about the dependencies.

If you think we should to that for the files in `_static` too then we can do that too.

Reply:
Now I got it, I think then that `COPY` is good there